pages/common/pycodestyle.md

```diff
@@ -0,0 +1,23 @@
+# pycodestyle
+
+> A tool to check Python code against PEP 8 style conventions.
+
+- Check the style of a single file:
+
+`pycodestyle {{file.py}}`
+
+- Check the style of a multiple files:
```

Comment:
The "a" should be removed (it appears to have been copied from the example above)